recipes/ckanweb-deploy-exts.rb

```diff
@@ -250,6 +253,18 @@
 		end
     end
 
+    if "#{pluginname}".eql? 'resource_visibility'
+        bash "Inject resource_visibility scheming preset if missing" do
+			user "#{account_name}"
+			cwd "#{config_dir}"
+			code <<-EOS
+				if [ -z "$(grep 'ckanext.resource_visibility:schema/presets.json' production.ini)" ]; then
+					sed -i "s/scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json$/scheming.presets = ckanext.scheming:presets.json ckanext.data_qld:presets.json ckanext.resource_visibility:schema/presets.json/g" production.ini;
```

Comment:
This won't play nicely with others wanting to override the presets. It should use a regex search and replace to append to the existing settings instead of replacing them.